webpack.config.js

```diff
@@ -110,8 +117,17 @@ var config = {
 
 
         ],
+
+        preLoaders: [
+            // All output '.js' files will have any sourcemaps re-processed by 'source-map-loader'.
+            {
+                test: /\.js$/,
+                loader: "source-map-loader"
+            }
+        ]
     },
     'postcss': [require('autoprefixer')],
+    'devtool': 'source-map',
```

Comment:
I followed the instructions [here](http://www.typescriptlang.org/docs/handbook/react-&-webpack.html).  Is this line not required?